.gitignore

```diff
@@ -28,7 +28,6 @@ project.xcworkspace
 
 # Android/IntelliJ
 #
-build/
```

Comment:
Why did you remove this line? Do we really want the build/ folder to be committed?

Reply:
It's dupilcated from the XCode section - line 7. Happy to add it back to make it explicit for Android.

Reply:
Maybe moving it to a "commons" section would also help ;) but you are right, it's still in there :)